crates/autopilot/src/run_loop.rs

```diff
@@ -145,6 +146,7 @@ impl RunLoop {
 
         // TODO: Keep going with other solutions until some deadline.
         if let Some(Participant { driver, solution }) = solutions.last() {
+            let timer = Metrics::get().solutions_processing_time.start_timer();
```

Comment:
Why is this PR not using the same style as the other time measurements we already have in this class? This is, instantiate a start time and call `Metrics::post_processed()`. Let's please stay consistent with existing patterns if we can or refactor everything but not have a lot of different styles in the same file.

Also, I think this starting point is wrong. It includes the roundtrip time of `reveal` which is not something that is "in our court". If a solver `sleeps` for 5s in their reveal implementation it shouldn't count towards post processing (post processing is all the aysnc DB operations that might take time on our end).